package.json

```diff
@@ -56,6 +56,7 @@
   ],
   "resolutions": {
     "ansi-regex": "5.0.1",
+    "ansi-html": "0.0.8",
```

Comment:
Force ansi-html to `0.0.8`. This is only used by the angular example dev kit server. So it should NOT affect anything else.